Scripts/build.py

```diff
@@ -156,7 +156,7 @@ def xcodebuild(scheme, workspace, config, commands):
 
 
 def check_string_output(command):
-    return subprocess.check_output(command).decode().strip()
+    return subprocess.check_output(command).decode('utf-8').strip()
```

Comment:
@tiennou python (2) just chokes at anything that is not ASCII if you don’t handle the encoding manually

Specifically I saw got messages like:

``
/Users/ssp/Developer/-open-projects/gitx/Classes/Views/PBRefMenuItem.m:104:74: warning: undeclared selector 'createTag:' [-Wundeclared-selector]
                [items addObject:[PBRefMenuItem itemWithTitle:@"Create Tag…" action:@selector(createTag:) enabled:YES]];
``

where the names of menu items with non-ASCII … got logged.